### PR TITLE
implemented case competition page lol cool

### PR DESCRIPTION
**Pull request by cindytung**

### CODE REVIEW

File: src/layouts/index.jsx [hunk 1]
```diff
@@ -7,8 +7,13 @@ import '../styles/base.scss';
 const TemplateWrapper = ({ children }) => (
   <div>
     <Navbar />
-    <div>
-      {children}
+    <div
+      style={{
+        margin: '0 auto',
+        paddingTop: 21,
```
Comment (dav1dn) [inline src/layouts/index.jsx]: would prefer this to be in a .scss file -- conventionally, you'd make a `$navbar-height` variable in _mixins.scss and then use that everywhere else you need to pad for it

File: src/styles/base.scss [hunk 2]
```diff
@@ -9,3 +9,55 @@
...
 }
+
+body {
+  font-family: 'Archer';
+}
+
+#city-scape-background {
+  flex-shrink: 0;
+  min-width: 100%;
+  min-height: 100%
```
Comment (dav1dn) [inline src/styles/base.scss]: ;

File: src/pages/case-competition.jsx [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
 
-export default class IRCJournal extends React.Component {
-
+export default class CaseCompetition extends React.Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="section">
+          <div className="section__header">
+            1st Annual IRC Case Competition
```
Comment (dav1dn) [inline src/pages/case-competition.jsx]: this is a page that should support multiple case competitions, right? you need to genericize case comp into components that can take arbitrary .md data to form each event, like a blog/list of events

File: src/pages/case-competition.jsx [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+import background from '../assets/images/city-scape-background.png';
 
-export default class IRCJournal extends React.Component {
-
+export default class CaseCompetition extends React.Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="section">
+          <div className="section__header">
```
Comment (dav1dn) [inline src/pages/case-competition.jsx]: use more descriptive element names -- this is a CaseCompetition, not a section

File: src/pages/case-competition.jsx [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+            politics. The case competition will engage UC Berkeley students on
+            their creativity and analytical skills in a format not commonly found
+            in international relations while also providing increased
+            professional exposure for Berkeley’s international relations programs
+            and the type of skills imparted on their students in an untraditional
+            geographic location for international relations and development
+            politics.
+          </div>
+        </div>
+        <img id="city-scape-background" alt="" src={background} />
```
Comment (dav1dn) [inline src/pages/case-competition.jsx]: think i prefer this as a background-image, but this is also fine

File: src/styles/base.scss [hunk 6]
```diff
@@ -9,3 +9,55 @@
...
+    text-align: center;
+    margin: 12px;
+    width: 100%;
+  }
+
+  .section__header {
+    font-size: 28px;
+  }
+
+  .section__subline {
```
Comment (dav1dn) [inline src/styles/base.scss]: subtext?

**Review by dav1dn — CHANGES_REQUESTED** [review]

File: src/styles/base.scss (hunk 6 above)
Comment (cindytung) [reply src/styles/base.scss]: english :~)

**Pull request merged**